src/azure-cli/azure/cli/command_modules/role/custom.py

```diff
@@ -331,8 +334,12 @@ def list_role_assignment_change_logs(cmd, start_time=None, end_time=None):  # py
                         else:
                             entry['scopeType'] = 'Resource'
 
-                    entry['roleDefinitionId'] = role_defs[payload['roleDefinitionId']][1]
-                    entry['roleName'] = role_defs[payload['roleDefinitionId']][0]
+                    # Look up the resource `name`, like b24988ac-6180-42a0-ab88-20f7382dd24c
+                    role_resource_name = payload['roleDefinitionId'].split('/')[-1]
+                    entry['roleDefinitionId'] = role_resource_name
```

Comment:
Who will use entry['roleDefinitionId'] later? Now it is like `b24988ac-6180-42a0-ab88-20f7382dd24c`, is it expected?

Reply:
This key has the same value as the original behavior:

```
d.id.split('/')[-1]
```

I didn't change the value.